ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -986,10 +994,11 @@ func (r *RayClusterReconciler) createWorkerPod(ctx context.Context, instance ray
 	}
 
 	if err := r.Create(ctx, &pod); err != nil {
+		r.Recorder.Eventf(&instance, corev1.EventTypeWarning, FailedToCreateResource, "Failed to create worker pod %s: %v", pod.Name, err)
```

Comment:
Can we include the namespace of the Pod too?

Reply:
Done!
I've also added in ns info for the other generated events.